[32mINFO    [0m test_logout:test_e2e.py:159 user has logged out